backend/src/backend/app.py

```diff
@@ -114,6 +128,36 @@ async def get_tasks(user_id: Annotated[str, Depends(get_user_id)]):
     del user_id
     return [{"id": 1, "title": "Buy Milk"}, {"id": 2, "title": "Buy Bread"}]
 
+# Open AI chat
+openai.api_key = OPENAI_API_KEY 
+
+# Pydantic models for request and response
+class ChatRequest(BaseModel):
```

Comment:
Python has an official guideline for code styling, called PEP8, and it recommends to have two blank lines between top-level functions/classes

https://peps.python.org/pep-0008/#blank-lines
> Surround top-level function and class definitions with two blank lines.

Reply:
not sure why `ruff` did not report this... maybe I made mistakes configuring the GitHub Actions.
From my experience, many teams request Python engineers to run `flake8` and `black` (code formatter) before committing changes, just for your information

Reply:
> run `flake8` and `black`

Thank you for letting me know, I will configure flake8 and black!